src/ROUTES.ts

```diff
@@ -236,6 +249,10 @@ const ROUTES = {
         route: 'r/:reportID/settings/room-name',
         getRoute: (reportID: string) => `r/${reportID}/settings/room-name` as const,
     },
+    REPORT_SETTINGS_GROUP_NAME: {
+        route: 'r/:reportID/settings/group-name',
```

Comment:
Unsure if there is a better url for this. Maybe `r/:reportID/settings/name` is better than `group-name` 🤔